src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1428,10 +1431,10 @@ codet java_bytecode_convert_methodt::convert_instructions(
 
       assert(arg0.id()==ID_virtual_function);
 
-      // does the function symbol exist?
+      // does the function symbol exist? is it virtual and inherited?
```

Comment:
This is where stubs get created if they are going to be created at all. We're suppressing creation of a stub when we can already see a callable implementation of the target virtual method.

Comment-wise the suggested comment is inaccurate-- how about
`// If we don't have a definition for the called symbol, and we won't inherit a definition from a superclass, create a stub.`